spec/amber/cli/commands/routes/routes_spec.cr

```diff
@@ -0,0 +1,69 @@
+require "../../../../spec_helper"
+require "../../../../support/helpers/cli_helper"
+
+require "cli/spec"
+
+include CLIHelper
+include Cli::Spec::Helper
+
+module Amber::CLI
+  extend Helpers
+  describe "amber routes" do
+    begin
+      context "in an `amber new` app with default options" do
+        cleanup
+        scaffold_app(TESTING_APP)
+        output = ""
+        ENV["CRYSTAL_CLI_ENV"] = "test"
+        MainCommand.run %w(routes) { |cmd| output = cmd.out.gets_to_end }
```

Comment:
Since these two lines (18 and 19) aren't required to run for the test at line 53, should these and the following 3 tests be wrapped in a describe?

Reply:
done